docs/lee_whitfield.md

```diff
@@ -2,7 +2,8 @@
 tags:
   -  People
 ---
-<img src="../assets/images/200px-LeeW.jpg" title="LeeW.jpg" width="200" alt="LeeW.jpg" align="right" />
+<img src="assets/images/200px-LeeW.jpg" title="LeeW.jpg" width="200" alt="LeeW.jpg" align="right" />
```

Comment:
This is a case where using `<img>` tag did make sense to me; I changed the path to have `../` prepended to make it render on the live site.